packages/react-conformance/src/defaultTests.tsx

```diff
@@ -0,0 +1,71 @@
+import { TestObject, TestingOptions } from './types';
+import { ComponentDoc } from 'react-docgen-typescript';
+import { mount } from 'enzyme';
+import * as React from 'react';
+import * as _ from 'lodash';
+import * as path from 'path';
+
+export const defaultTests: TestObject = {
+  /** Component has a docblock with 5 to 25 words */
+  'has-docblock': (componentInfo: ComponentDoc, testInfo: TestingOptions) => {
+    const maxWords = 25;
+    const minWords = 5;
+
+    // No need to check if the description is undefined, ComponentDoc.description is a "string" not "string | undefined"
+    it(`has a docblock with ${minWords} to ${maxWords} words`, () => {
+      expect(_.words(componentInfo.description)).toBeGreaterThanOrEqual(minWords);
+      expect(_.words(componentInfo.description)).toBeLessThanOrEqual(maxWords);
+    });
+  },
+
+  /** Component file exports a valid React element type  */
+  'exports-component': (componentInfo: ComponentDoc, testInfo: TestingOptions) => {
+    it(`exports component from file under correct name`, () => {
+      const componentFile = require(testInfo.componentPath);
+      const constructorName = testInfo.Component.prototype.constructor?.name;
+      if (testInfo.useDefaultExport) {
+        expect(componentFile.default).toBe(testInfo.Component);
+      } else {
+        expect(componentFile[constructorName]).toBe(testInfo.Component);
+      }
+    });
+  },
+
+  /** Component file exports a valid React element and can render it */
+  'component-renders': (componentInfo: ComponentDoc, testInfo: TestingOptions) => {
```

Comment:
why are we using kebab case here? I'm a little confused on the usage.

Reply:
I used kebab casing to make it similar to how eslint names its rules. Also at the beginning I was thinking on how to be able to disable rules in case they can be opted out and using something like: disableRules: ['component-renders', 'next-rule'] would be a nice way of doing it.